doc/kak.1

```diff
@@ -1,251 +1,282 @@
-.BR \-ro
-enter in \fIreadonly mode\fR, all the buffers opened will not be written to disk
-
-.TP
-.BR +line "[:" column "]"
-specify a target \fIline\fR and \fIcolumn\fR for the first file; when the
-plus sign is followed by only a colon, then the cursor is sent to the last
-line of the file
-
-.TP
-.BR file
-one or more \fIfile\fRs to edit
-
-.SH ENVIRONMENT
-
-.TP
-.BR KAKOUNE_POSIX_SHELL
-Overrides the posix shell binary path to use for \fI%sh{...}\fR expansion.
-
-.TP
-.BR KAKOUNE_CONFIG_DIR
-Overrides the location of the directory containing kakoune user configuration,
-defaults to \fI$XDG_CONFIG_HOME/kak\fR if unset.
-
-.TP
-.BR XDG_CONFIG_HOME
-Path to the user configuration directory, defaults to \fI$HOME/.config\fR if unset.
-
-.TP
-.BR XDG_RUNTIME_DIR
-Path to the user session sockets, defaults to \fI$TMPDIR/kakoune\fR if unset.
-
-.SH FILES
-
-In the paths documented below, \fI<rtdir>\fR refers to the runtime directory,
-whose value is determined in relation to the path to the \fBkak\fR binary:
-\fI<rtdir>\fR = \fI<path_to_kak_binary>/../share/kak\fR.
-
-If not started with the \fB\-n\fR switch, Kakoune will first load
-\fI<rtdir>/kakrc\fR, which will in turn load the following additional files:
-
-.nf
-.RS
-if the \fI$KAKOUNE_CONFIG_DIR/autoload\fR directory exists, recursively load every
-\fI*.kak\fR files in it and its sub-directories
-.RE
-.fi
-
-.nf
-.RS
-if it does not exist, fall back to the system\-wide autoload directory
-in \fI<rtdir>/autoload\fR, and recursively load all files in a similar way
-.RE
-.fi
-
-.nf
-.RS
-\fI<rtdir>/kakrc.local\fR, if it exists; this is a user\-defined system\-wide
-configuration
-.RE
-.fi
-
-.nf
-.RS
-\fI$KAKOUNE_CONFIG_DIR/kakrc\fR, if it exists; this is the user configuration
-.RE
-.fi
-
-Consequently, if the \fI$KAKOUNE_CONFIG_DIR/autoload\fR directory exists,
-only scripts stored within that directory will be loaded \- the built-in
-\fI*.kak\fR files \fBwill not be\fR.
-
-Users who still want to have the built\-in scripts loaded along their own
-can create a symbolic link to \fI<rtdir>/autoload\fR (or to individual
-scripts in it) in their user\-configuration directory:
-
-.nf
-.RS
-ln -s \fI<rtdir>\fR/autoload "${XDG_CONFIG_HOME:-$HOME/.config}"/kak/autoload
-.RE
-.fi
-
-.SH EXAMPLES
-
-.PP
-Edit a file:
-
-.nf
-.RS
-kak /path/to/file
-.RE
-.fi
-
-.PP
-Edit multiple files (multiple buffers will be created):
-
-.nf
-.RS
-kak ./file1.txt /path/to/file2.c
-.RE
-.fi
-
-.PP
-Insert a modeline that sets the tabstop variable at the beginning of several
-source code files:
-
-.nf
-.RS
-kak \-f "ggO// kak: tabstop=8<esc>" *.c
-.RE
-.fi
-
-.SH SEE ALSO
-
-.BR vi (1),
-.BR vim (1),
-.BR sam (1plan9)
+.Dd June 15, 2021
+.Dt KAK 1
+.Os
+.Sh NAME
+.Nm kak
+.Nd a vim-inspired, selection oriented code editor
+.
+.Sh SYNOPSIS
+.Nm
+.Op Fl d
+.Op Fl n
+.Op Fl ro
+.Op Fl c Ar session_id | Fl s Ar session_id
+.Op Fl ui Ar ui_type
+.Op Fl e Ar command
+.Op Fl E Ar command
+.Op Ar +line Ns Oo Ar :column Oc | +:
```

Comment:
The plus sign and the colon should be bold since they need to be typed literally: `.Op Sy + Ns Ar line Ns Oo Sy \&: Ns Ar column Oc | Sy +:`. mdoc(7) has this to say about the .Sy macro: “When none of the semantic macros fit, it is also adequate for syntax elements that have to be given or that appear verbatim.” The `\&` inserts a zero-width space used to escape the colon, which is treated as a delimiter otherwise.